### PR TITLE
Making sure profiler.oql tests pass.

### PR DESCRIPTION
**Pull request by JaroslavTulach**
Adding a heap dump file omitted on the Apache donation. Enabling testing as part of the `scripting.sh` travis script.

### CODE REVIEW

File: profiler/profiler.oql/test/unit/src/org/netbeans/modules/profiler/oql/engine/api/impl/OQLEngineTest.java [hunk 1]
```diff
@@ -515,6 +516,7 @@ public boolean visit(Object o) {
                 return false;
             }
         });
+        output[0] = output[0].replaceAll("UID=301077366599181[0-9]*", "UID=301077366599181567");
```
Comment (matthiasblaesing) [inline profiler/profiler.oql/test/unit/src/org/netbeans/modules/profiler/oql/engine/api/impl/OQLEngineTest.java]: Could you please add a comment what this making replacement does? Not so much for the change itself, as for future readers, that wonder what was this meant to say.

Comment (JaroslavTulach) [reply profiler/profiler.oql/test/unit/src/org/netbeans/modules/profiler/oql/engine/api/impl/OQLEngineTest.java]: According to @thurka there was some bug in the serialization that affects the `UID` value. Tomáši, can you share the JDK bug reference?

File: profiler/profiler.oql/test/unit/src/org/netbeans/modules/profiler/oql/engine/api/impl/OQLEngineTest.java [hunk 2]
```diff
@@ -50,6 +50,7 @@ public OQLEngineTest() {
 
     @BeforeClass
     public static void setUpClass() throws Exception {
+        System.setProperty("polyglot.js.nashorn-compat", "true");
```
Comment (JaroslavTulach) [inline profiler/profiler.oql/test/unit/src/org/netbeans/modules/profiler/oql/engine/api/impl/OQLEngineTest.java]: This magic option will be moved to `Bindings` [once this change](https://github.com/graalvm/graaljs/commit/af68dca48fa16f7542e6f363ca8099d02a3d8ccd) gets in.

**Comment by JaroslavTulach** [comment]
> add is a licenseinfo.xml file, so that the license info is still present 

Done in b4cdd1b71995da60355185724bb4a684f18feb49

File: profiler/profiler.oql/test/unit/src/org/netbeans/modules/profiler/oql/engine/api/impl/OQLEngineTest.java (hunk 1 above)
Comment (thurka) [reply profiler/profiler.oql/test/unit/src/org/netbeans/modules/profiler/oql/engine/api/impl/OQLEngineTest.java]: https://netbeans.org/bugzilla/show_bug.cgi?id=256761. JDK bug is already fixed, there is no need for workaround.

**Comment by JaroslavTulach** [comment]
The two failing checks are unrelated to `profiler.oql` changes. Merging anyway. CCing @jlahoda

**Pull request merged**